release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -650,6 +650,37 @@ If you disable the `ImageRegistry` capability or if you disable the integrated {
 
 For more information, see xref:../nodes/pods/nodes-pods-secrets.adoc#auto-generated-sa-token-secrets_nodes-pods-secrets[Automatically generated secrets].
 
+[discrete]
+[id="ocp-4-15-ovnic-default-range"]
+==== Open Virtual Network Infrastructure Controller default range
+
+Previously, the IP address range `168.254.0.0/16` was the default IP address range that the Open Virtual Network Infrastructure Controller used for the transit switch subnet. With this update, the Controller uses `100.88.0.0/16` as the default IP address range. Do not use this IP range in your production infrastructure network. (link:https://issues.redhat.com/browse/OCPBUGS-20178[*OCPBUGS-20178*])
+
+[discrete]
+[id="ocp-4-15-egress-ip-azure-private-cluster"]
+==== Application of egress IP addresses to egress nodes on private Microsoft Azure clusters
+
+There is a limitation in private Microsft Azure clusters where secondary IPs addresses designated as egress IP addresses lack outbound connectivity. This means that pods associated with these IP addresses are unable to access the internet. However, they can still reach external servers within the infrastructure network, which is the intended use case for egress IP addresses. This update enables egress IP addresses for Microsoft Azure setups, allowing outbound connectivity to be achieved through outbound rules. (link:https://issues.redhat.com/browse/OCPBUGS-5491[*OCPBUGS-5491*])
```

Comment:
```suggestion
There is a limitation in private Microsoft Azure clusters where secondary IPs addresses designated as egress IP addresses lack outbound connectivity. This means that pods associated with these IP addresses are unable to access the internet. However, they can still reach external servers within the infrastructure network, which is the intended use case for egress IP addresses. This update enables egress IP addresses for Microsoft Azure clusters, allowing outbound connectivity to be achieved through outbound rules. (link:https://issues.redhat.com/browse/OCPBUGS-5491[*OCPBUGS-5491*])
```

Reply:
This seems like KI rather than a  Notable technical change.

Reply:
Actually moved it back to bugs. It was resolved though, but after reading back through the Jira ticket felt it was more in line with a bug.